views/park_views.py

```diff
@@ -48,3 +48,65 @@ def get_single_park(id):
     park = Parks(data['id'], data['name'], data['history'], data['city'], data['state'], data['latitude'], data['longitude'])
 
     return park.__dict__
+
+def create_park(park):
+    """creating a post request to create a park. Checks if all values are included in post, returns error"""
+    with sqlite3.connect("./national_park.sqlite3") as conn:
```

Comment:
Recommend using the create_resource function from sql_helper.py

Reply:
Ah! I'll work on modifying in my next bunch of changes. Not enough brain power to make those changes right now and try to work it out the error message in the event of missing info.